modules/MyButton.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import {TouchableOpacity, Text, StyleSheet} from 'react-native';
+
+class MyButton extends Component{
```

Comment:
If you don't need state then don't create class for that component.
You can just create and function - its argument will be props from the parents.

Reply:
This is called - stateless component.